docs/astro.config.mjs

```diff
@@ -84,12 +84,12 @@ export default defineConfig({
             {
               label: 'Showcase',
               link: 'showcase',
-              // translations: {
-              //   es: '',
-              //   ja: '',
-              //   fr: '',
-              //   it: '',
-              // },
+              translations: {
+                es: '',
+                ja: 'ショーケース',
+                fr: '',
+                it: '',
```

Comment:
This works because the empty string is being ignored, but I might suggest commenting these out just to be explicit:

```suggestion
                // de: '',
                // es: '',
                ja: 'ショーケース',
                // fr: '',
                // it: '',
```